public/styles.css

```diff
@@ -31,9 +31,23 @@
   margin-left: 25px;
 }
 
+.carousel-navbar-bin {
+  margin-right: 10px;
+  margin-top: 10px;
+}
+
 .carousel-navbar {
-  margin-right: 5px;
-  text-align: right;
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
Is this a particular reason to use 'flex-direction:row-reverse' instead of 'justify-content: flex-end'? I am asking because row-reverse will also change the order of the dots. Just want to make sure that's what you want.

Reply:
For some reason, one of the dots would consistently disappear when using 'justify-content: flex-end' (or the dots wouldn't move over) so I changed the flex-direction to row-reverse.
You bring up a really good point about the order of the dots - I am going to transition everything to styled components (and see if it will respond as expected to 'justify-content...') before coding the behavior of the dots (color-change to red).
Thank you! :)

Reply:
Interesting. I wonder why. Looking forward to the styled components transition.